Makefile

```diff
@@ -0,0 +1,49 @@
+PACKER_CMD ?= packer
+RELEASE ?= alpha
+VERSION ?= current
+DIGEST_URL ?= https://$(RELEASE).release.flatcar-linux.net/amd64-usr/current/flatcar_production_iso_image.iso.DIGESTS
+CONFIG ?= flatcar-linux-config.yml
+DISK_SIZE ?= 40000
+MEMORY ?= 2048M
+BOOT_WAIT ?= 45s
+CT_DOWNLOAD_URL ?= https://github.com/coreos/container-linux-config-transpiler/releases/download
+CT_VER ?= v0.7.0
+ARCH ?= $(shell uname -m)
+HEADLESS ?= false
```

Comment:
Why did you decide to default to `false`? IMHO it should be `true`.

Reply:
This is because of the assumption that this would be run on bare computers where it is okay to run in non-headless mode, and as far as I have tested, it does not do well in headless mode.

Reply:
I'd prefer to keep `HEADLESS ?= false` by default. From the packer build's perspective, it's quite more intuitive to watch the whole VNC output. Otherwise the output is quite opaque, so that sometimes users don't have any clue what's going on. Unfortunately packer builds fail from time to time without any reason, so the VNC output is indeed helpful for debugging.

Reply:
OK.